app/src/app/routes/Sources/Sources.tsx

```diff
@@ -12,10 +12,10 @@ const useStyles = makeStyles((theme) => ({
   },
   gridContainer: {
     flexGrow: 1,
-    paddingTop: theme.spacing(5),
+    paddingTop: theme.spacing(2),
   },
   container: {
-    padding: theme.spacing(0, 5),
+    padding: theme.spacing(5, 8),
   },
```

Comment:
You can also get rid of it

Reply:
It's making a padding, without it it looks like this :
![image](https://user-images.githubusercontent.com/47388675/130527605-1829baaa-d1c4-4d12-a5f8-bf0dae17a5b4.png)